pkg/volume/volume.go

```diff
@@ -28,6 +28,32 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	Accountable
+}
+
+type VolumeDefaults struct {
+	AccountingNil
+}
+
+type Accountable interface {
+	GetAccounting() (*Accounting, error)
+}
+
+const unknownSize = -1
+
+type Accounting struct {
+	PodBytesUsed    int
+	SharedBytesUsed int
```

Comment:
PodBytesUsed is the bytes used by the Pod - calculated here by running du -s on the directory
SharedBytesUsed is the bytes used on the actual device.  Perhaps this isn't useful or is something we won't want to support long term and I should remove it.  Thoughts?